examples/Scikit.ipynb

```diff
@@ -0,0 +1,330 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "888638fa",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "The purpose of this example is to demonstrate the API's ability to work with a variety of machine learning libraries.\n",
+    "Here we use scikit learn's `LogisticRegression` class to implement a logistic regression model for image classification of the MNIST dataset.\n",
+    "This is inspired and partially taken from thie example of scikit's website:<br>\n",
+    "&emsp;&emsp;https://scikit-learn.org/stable/auto_examples/linear_model/plot_sparse_logistic_regression_mnist.html\n"
```

Comment:
I think it would be good to add more to this introduction, specifically about what we are adding to this example on top of the reference example from within scikit-learn.

For example, the example shows, effectively, an attention visualization, as we are duplicating here. The visualization of saliency heatmaps below seems very similar. I believe it is insinuated in the "Calling the Application" section, but we could be explicit here stating that the Scikit-Learn example visualization relies on *intrinsics* of the model in order to determine the "attention" visualization (whitebox), however our API's approach is classifier agnostic, "blackbox", and shows very comparable results without knowing anything about the specific classifier model being used, though likely some amount more processing intensive.